velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2858,6 +2866,17 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
 
+TEST_F(DateTimeFunctionsTest, currentTimeTest) {
+  auto emptyRowVector = makeRowVector(ROW({}), 1);
+  auto tz = "America/Los_Angeles";
+  setQueryTimeZone(tz);
+  auto currentTimeBefore = getCurrentTime(tz);
```

Comment:
Nit : Call this variable before is sufficent.

Reply:
Done